src/insurance-policy/insurance-policy.resource.ts

```diff
@@ -27,8 +26,9 @@ const mapInsurancePolicyProperties = (policy: InsurancePolicy): MappedInsuranceP
     gender: policy.owner.person.gender,
     age: policy.owner.person.age,
     patientName: policy.owner.person.preferredName.display,
-    insurance: '--',
+    insurance: policy.insurance.name,
     birthdate: formatDate(new Date(policy.owner.person.birthdate)),
+    insurancePolicyNo: policy.insurancePolicy?.insurancePolicyId ?? '--',
```

Comment:
Isn't it supposed to be just policy.insurancePolicyId?

Reply:
There will be no insurancePolicy object just like insurance ?

Reply:
<img width="1718" alt="Screenshot 2025-05-20 at 17 02 12" src="https://github.com/user-attachments/assets/f87f52cb-b71f-46f5-93d2-194d12d6aab1" />